src/radical/pilot/raptor/master.py

```diff
@@ -328,7 +334,7 @@ def submit_workers(self, descr: Dict[str, Union[str, int]],
                 task['origin']            = 'raptor'
                 task['description']       = TaskDescription(td).as_dict()
                 task['state']             = rps.AGENT_STAGING_INPUT_PENDING
-                task['status']            = 'NEW'
+                task['status']            = self.NEW
```

Comment:
sorry, missed this one, should it take constant value from `rp.states`? (even tho it has the same value as this one)

Reply:
nevermind, looks right!